src/html.pre.js

```diff
@@ -190,6 +190,7 @@ async function pre(payload, action) {
     } else {
       logger.debug('html-pre.js - No REPO_RAW_ROOT provided');
     }
+    return payload;
```

Comment:
this is no longer needed with the new pipeline. and even discouraged.

Reply:
I do not know if this is discouraged or not, but in any case, this is out of the scope of the issue you are fixing. Why taking the risk to break something ? ;)

Reply:
I meant: @stefan-guggisberg why did you add it here?

- it is not relevant to the secrets mentioned in this PR
- it should not be added again, since it's no longer needed.

Reply:
I added the line after the build failed on CI:
```
/home/circleci/project/src/html.pre.js
  148:16  error  Expected to return a value at the end of async function 'pre'  consistent-return
``` 

@tripodsan Your earlier commit also failed on CI but apparently you didn't notice:
https://circleci.com/gh/adobe/project-helix.io/177?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

Reply:
> I meant: @stefan-guggisberg why did you add it here?
> it is not relevant to the secrets mentioned in this PR

the commit msg explained it: `build(lint) fix CI build`

Reply:
hmmm. then we need to fix the tests. we don’t need a return value anymore. it will be discarded in the future.

Reply:
It was this commit that caused the CI build failure: https://github.com/adobe/project-helix.io/commit/8c72b83e665d6a4ce664b8704850b7abd2ac72f4#diff-2ee4b2405528090b05a27eaae296376fL194

Reply:
@tripodsan any objection to merge this PR?

Reply:
In general, I would not alter files that don't belong to the topic of the pull request. but, go ahead.
also, just updating the secrets doesn't need a PR IMO :-)

what I don't understand is, since I fixed the smoke tests which use project-helix.io for testing, I don't see why the return value change throws your tests off... you probably don't use the latest helix-cli ....

but go ahead, merge the PR, we look at the return value later :-)

Reply:
The smoke tests do not include the `project-helix.io` unit tests and linting validation, that's why we do not see it. This would not make sense. 
Here, the issue has been introduced by this commit to `master`: https://github.com/adobe/project-helix.io/commit/8c72b83e665d6a4ce664b8704850b7abd2ac72f4 which is flagged as red.

I think it is ok to merge with the change to fix the linting. And from now, no more direct commits to master :)

Reply:
> The smoke tests do not include the project-helix.io unit tests and linting validation,

ah right: I was confused. they execute the helix-cli tests:
https://github.com/adobe/helix-continuous/blob/f3b0ea64be92b91c027f15d8de31edd2484366cd/.circleci/config.yml#L88-L93

Reply:
> In general, I would not alter files that don't belong to the topic of the pull request. but, go ahead.

Well, somebody broke the build ;-) ...which caused my PR to fail on CI Build. Since I wanted my PR to pass CI Build I fixed the build failure in the PR.